web_src/js/features/common-issue.js

```diff
@@ -2,15 +2,25 @@ import $ from 'jquery';
 import {updateIssuesMeta} from './repo-issue.js';
 
 export function initCommonIssue() {
-  $('.issue-checkbox').on('click', () => {
-    const numChecked = $('.issue-checkbox').children('input:checked').length;
-    if (numChecked > 0) {
+  const checkboxOperate = () => {
+    if ($('.issue-checkbox input').is(':checked')) {
       $('#issue-filters').addClass('hide');
       $('#issue-actions').removeClass('hide');
+      $('#issue-actions .six').prepend($('.issue-checkbox-all'));
     } else {
       $('#issue-filters').removeClass('hide');
       $('#issue-actions').addClass('hide');
+      $('#issue-filters .six').prepend($('.issue-checkbox-all'));
```

Comment:
What's the purpose of these two `prepend($('.issue-checkbox-all'));`?

Reply:
If the switch is checked, move the Check All checkbox in the template to the question filter or action area

Reply:
Just tried, the layout here was quite interesting before. It's right!